traffic_portal/app/src/common/modules/table/deliveryServiceServers/TableAssignDSServersController.js

```diff
@@ -17,87 +17,54 @@
  * under the License.
  */
 
-var TableAssignDSServersController = function(deliveryService, servers, assignedServers, $scope, $uibModalInstance) {
-
-	var selectedServers = [];
+/** @typedef { import('../../../../common/modules/table/agGrid/CommonGridController').CGC } CGC */
 
-	var addAll = function() {
-		markVisibleServers(true);
-	};
-
-	var removeAll = function() {
-		markVisibleServers(false);
-	};
+var TableAssignDSServersController = function(deliveryService, servers, assignedServers, $scope, $uibModalInstance) {
 
-	var markVisibleServers = function(selected) {
-		var visibleServerIds = $('#dsServersUnassignedTable tr.server-row').map(
-			function() {
-				return parseInt($(this).attr('id'));
-			}).get();
-		$scope.servers = _.map(servers, function(server) {
-			if (visibleServerIds.includes(server.id)) {
-				server['selected'] = selected;
-			}
-			return server;
-		});
-		updateSelectedCount();
-	};
+	$scope.selectedServers = [];
 
-	var updateSelectedCount = function() {
-		selectedServers = _.filter($scope.servers, function(server) { return server['selected'] == true; } );
-		$('div.selected-count').html('<b>' + selectedServers.length + ' servers selected</b>');
-	};
+	/** @type CGC.ColumnDefinition */
+	$scope.columns = [
+		{
+			headerName: "Host",
+			field: "hostName",
+			checkboxSelection: true,
+			headerCheckboxSelection: true,
+		},
+		{
+			headerName: "Cache Group",
+			field: "cachegroup",
+		},
+		{
+			headerName: "Profile",
+			field: "profile"
+		}
+	];
 
 	$scope.deliveryService = deliveryService;
 
-	$scope.servers = _.map(servers, function(server) {
-		var isAssigned = _.find(assignedServers, function(assignedServer) { return assignedServer.id == server.id });
+	$scope.servers = servers.map(server => {
+		let isAssigned = assignedServers.find(assignedServer => assignedServer.id === server.id);
 		if (isAssigned) {
 			server['selected'] = true;
 		}
 		return server;
 	});
 
-	$scope.selectAll = function($event) {
-		var checkbox = $event.target;
-		if (checkbox.checked) {
-			addAll();
-		} else {
-			removeAll();
-		}
-	};
-
-	$scope.onChange = function() {
-		updateSelectedCount();
-	};
-
 	$scope.submit = function() {
-		var selectedServerIds = _.pluck(selectedServers, 'id');
+		const selectedServerIds = this.selectedServers.map(s => s["id"]);
```

Comment:
Since when does our build system allow fat-arrow functions? I had to try it myself to believe it worked - that used to break in half.

Reply:
/shrug